godtools/App/Features/Menu/Domain/UseCases/GetUserActivityStatsUseCase/GetUserActivityStatsUseCase.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  GetUserActivityStatsUseCase.swift
+//  godtools
+//
+//  Created by Rachael Skeath on 3/20/23.
+//  Copyright © 2023 Cru. All rights reserved.
+//
+
+import Foundation
+import GodToolsToolParser
+import SwiftUI
+
+class GetUserActivityStatsUseCase {
+    
+    private let localizationServices: LocalizationServices
+    
+    init(localizationServices: LocalizationServices) {
+        
+        self.localizationServices = localizationServices
+    }
+    
+    func getUserActivityStats(from userActivity: UserActivity) -> [UserActivityStatDomainModel] {
+        
+        let toolOpensStat = UserActivityStatDomainModel(
+            iconImageName: "tool-opens",
+            text: localizationServices.stringForMainBundle(key: "account.activity.toolOpens"),
+            textColor: Color(red: 5 / 255, green: 105 / 255, blue: 155 / 255),
```

Comment:
For creating a Color with RGB can you create a ```Color extension``` for this. 
I had created similar logic ```getColorWithRGB``` in ```ColorPalette.swift```, but it would be more clean to put this in an extension as ColorPalette pertains to colors that are stored in the assets catalog.